### PR TITLE
Unify test_run_transformers* and test_eval_model* scripts

### PR DESCRIPTION
**Pull request by OliviaLynn**
1. Unifies:
  - all combinations of `test_run_transfomers.py` (`_DC2.py`, `_DC2_redshift.py`, etc) into `any_test_run_transformers.py`
  - all combinations of test_eval_model.py into any_test_eval_model.py

2. Combinations can be accessed via new `--use-dc2` and `--use-redshift` flags.

3. `$bash run_all.sh` runs all combinations (including swin/mvitv2) and outputs to `run_all.log`

Other:
- Removing old, non-solo scripts as well.
- Skipping train_* scripts, as these do not run as-is at the moment.
- Planning to follow this with a PR to pull some of the config values out into the config files.
- For #17

### CODE REVIEW

File: src/deepdisc/astrodet/astrodet.py [hunk 1]
```diff
@@ -99,6 +99,7 @@
 from tabulate import tabulate
 from torch.nn.parallel import DistributedDataParallel
 
+from deepdisc.astrodet import detectron as detectron_addons
```
Comment (drewoldag) [inline src/deepdisc/astrodet/astrodet.py:102]: Not a blocker - I'm not immediately seeing why this line was added.

**Review by drewoldag — APPROVED** [review]
This all looks pretty reasonable to me. Thanks for putting it together!

File: src/deepdisc/astrodet/astrodet.py (hunk 1 above)
Comment (OliviaLynn) [reply src/deepdisc/astrodet/astrodet.py:102]: Wasn't sure if I should explicitly mention it (nice eye there)! 

It should have been added in the process of combining various versions of `train_mapper_cls` and moving it into astrodet. The previous versions had all lived in places that had this import, and I had overlooked that in the astrodet PR.

**Review by grantmerz — COMMENTED** [review]
I think rather than using conditional statements in the script it would be better to place these conditional variables in a solo config.  Mostly to better keep track of changes and minimize the clutter of the actual train script (I now see you've planned this in a follow-up PR)

**Pull request merged**